### DISCUSSION TITLE
Postgres as data source?

### DISCUSSION THREAD

**Discussion by mieciu** [zendesk/maxwell#2048] [Ideas]
Hi,

I was just wondering, are there any plans to introduce Postgres support in the project?
Or maybe there are some blocking architectural concepts/assumptions that raise the red flag.

Overall, I really appreciate simplistic setup and approach to problem (no extensions) presented here and I think it would be a very good idea for maxwell to act as more generic solution, rather than MySQL-> Kafka data pump.

**Comment by osheroff** [comment]
hi, there are no plans to introduce postgres support.  I have no idea what the scope of the effort would be, but I'd imagine it's in months.  So unless someone is donating big chunks of code, probably won't happen.

**Comment by dingguitao** [comment]
FYI.

https://github.com/confluentinc/bottledwater-pg

**Comment by smferguson** [comment]
followup fyi, the current maintainer doesn't think it's prod-ready yet:
https://github.com/confluentinc/bottledwater-pg/issues/96

he specifically cites these issues.
https://github.com/confluentinc/bottledwater-pg/issues/36
https://github.com/confluentinc/bottledwater-pg/issues/99

it may still be suitable for some use cases

**Comment by wushujames** [comment]
FYI, the Debezium project is also working on Postgres->Kafka. Theirs is also incomplete. http://debezium.io/docs/connectors/postgresql/
